app/lib/package.rb

```diff
@@ -40,7 +42,8 @@ def build_zip
 
     Zip::File.open("#{@directory}/#{@translation.zip_name}", Zip::File::CREATE) do |zip_file|
       add_pages(zip_file, manifest)
-      add_attachments(zip_file, manifest)
+      add_tips(zip_file, manifest)
+      #add_attachments(zip_file, manifest)
```

Comment:
`add_attachments` should still run, guessing you commented it out for local testing